xla/service/gpu/backend_configs.proto

```diff
@@ -260,4 +260,10 @@ message GpuBackendConfig {
 
     CudnnfMHABackendConfig cudnn_fmha_backend_config = 9;
   }
+
+  // This attribute instructs the latency-hiding scheduler to delay
+  // scheduling this particular instruction. Note that setting this to
+  // true will make this instruction scheduled at the very beginning of
+  // the parent computation before every other nodes.
+  bool should_force_delay_schedule = 10;
```

Comment:
Maybe just "force_delay_schedule"? Or "forced_schedule_last"? "should" leaves too much room to interpretation and I think we could be more self-descriptive.

Additionally, "force_delay" (at least to me) contradicts the description of "makes this instruction scheduled at the very beginning", maybe "force_schedule_first"?

I think a motivating use-case should also be mentioned in the comment.

Reply:
Yea the name could be confusing. LHS is doing a bottom-up scheduling, so schedule last in LHS is the same as schedule first in execution order. I will change this to `force_earliest_schedule` then.